index.bs

```diff
@@ -4100,7 +4100,7 @@ Attributes</h4>
 <dl dfn-type=attribute dfn-for="AnalyserNode">
 	: <dfn>fftSize</dfn>
 	::
-		The size of the FFT used for frequency-domain analysis.
+		The size of the FFT used for frequency-domain analysis (in frames).
```

Comment:
The spec generally uses "sample-frames" for this.  Don't know why. I always just say frames.